app/browser/api/ledger.js

```diff
@@ -1117,6 +1119,36 @@ const pageDataChanged = (state, viewData = {}, keepInfo = false) => {
   return state
 }
 
+const checkBrowserActivityTime = (state) => {
+  // Fuzzing threshold
+  const minTime = 30 * ledgerUtil.milliseconds.minute
+  const publisherKeys = Object.keys(synopsis.publishers)
```

Comment:
We should be getting publishers from the state and not synopsis

Reply:
done